content/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository.md

```diff
@@ -25,7 +25,7 @@ versions:
 
 {% data reusables.pull_requests.automatically-delete-branches %}
 
-If the branch you want to delete is the repository's default branch, you must choose a new default branch before deleting the branch. For more information, see "[Changing the default branch](/github/administering-a-repository/changing-the-default-branch)."
+#### If the branch you want to delete is the repository's default branch, you must choose a new default branch before deleting the branch. For more information, see "[Changing the default branch](/github/administering-a-repository/changing-the-default-branch)."
```

Comment:
While the formatting change you've made makes the text stand out, it turns this paragraph into a heading which makes the spacing a little weird. I think that it would be better to convert it into a [note callout](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#callout-tags). Like the one towards the top of this page: [Creating a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).

Something like:

```suggestion
{% note %}

**Note:** If the branch you want to delete is the repository's default branch, you must choose a new default branch before deleting the branch. For more information, see "[Changing the default branch](/github/administering-a-repository/changing-the-default-branch)."

{% endnote %}
```

Reply:
Yes, that does seem like a better alternative. It looks cleaner that way.

Reply:
Thanks. You can see what it will look like here: https://docs-1271--patch-1.herokuapp.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository#deleting-a-branch (although that staging deployment will disappear when we merge this PR).